mit-bitcoin-expo/mit-bitcoin-expo-2022/lightning-panel.md

```diff
@@ -12,20 +12,20 @@ date: 2022-07-05
 Jeremy Rubin: 00:00:07
 
 I was hoping that we can have our panelists, and maybe we'll start with Rene.
-Just introduce themselves quickly, and also answer the question, What is Lightning to you?
+Just introduce themselves quickly, and also answer the question, what is `Lightning` to you?
```

Comment:
We need to define better the usage of backticks. It's apparent from your reviews and the reviews of other that is not clear when to use backticks and when to not use.
But even if that's the case, you are not consistent with your usage of backticks - you are using them here but not in other parts of the transcript.

Please remove all the backticks that you are using and use backticks only when referring to [opcodes](https://en.bitcoin.it/wiki/Script#Opcodes)